app/lib/account/consent_backend.dart

```diff
@@ -174,3 +172,16 @@ abstract class ConsentAction {
   /// Callback on rejecting the consent or timeout.
   Future onDelete(String userId, List<String> args);
 }
+
+/// Callbacks for publisher member consents.
+class _PublisherMemberAction implements ConsentAction {
```

Comment:
What if we give `ConsentAction` a constructor that takes two callbacks... then we won't have to create so many classes...


```dart
class ConsentAction {
  final Future<void> Function(String userId, List<String> args) onAccept;
  ...

  ConsentAction(this.onAccept, ...);
}
```